ts/src/base/Exchange.ts

```diff
@@ -3049,7 +3049,7 @@ export default class Exchange {
                 // find lowest precision (which is more desired)
                 const precision = this.safeString (network, 'precision');
                 const precisionMain = this.safeString (currency, 'precision');
-                if (precisionMain === undefined || Precise.stringLt (precision, precisionMain)) {
+                if (precisionMain === undefined || Precise.stringGt (precision, precisionMain)) {
```

Comment:
we should only re-set if current network's precision (in networks loop) is higher than previous.